collections/Auct_E/MS_Auct_E_2_19.xml

```diff
@@ -34,32 +34,68 @@
                      <idno type="SCN">3016</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>
+                  <hi rend="smallcaps">Book of Job with catena compilation</hi>Incomplete catena compiled, possibly by the scribe, from the types known as Γ and w, with material taken from a hitherto unidentified source.</head>
                <msContents>
                   <msItem class="#biblia" n="1" xml:id="MS_Auct_E_2_19-item1">
-                     <title key="work_10835" type="desc">Catena on Job.</title>
+                     <locus>fols. 1r-75v</locus>
+                     <title key="work_10835" type="desc">Catena compilation commenting selected passages of the Book of Job.</title>
```

Comment:
it is a minor point, but the Bodleian convention is to enclose in parentheses (fols. 1r-75v)